cli/providers/providers.go

```diff
@@ -318,35 +318,22 @@ func attachFlags(flagset *pflag.FlagSet, flags []plugin.Flag) {
 	}
 }
 
-func getFlagValue(flag plugin.Flag, cmd *cobra.Command) *llx.Primitive {
+func getFlagValue(flag plugin.Flag) *llx.Primitive {
 	switch flag.Type {
 	case plugin.FlagType_Bool:
-		v, err := cmd.Flags().GetBool(flag.Long)
```

Comment:
Do we need a bind equivalent somewhere? Like in https://github.com/mondoohq/cnspec/blob/main/apps/cnspec/cmd/scan.go#L105

Reply:
We already have it for all flags inside a provider. Though you're pointing to cnspec code so I'm assuming you want to know if this will work there. I can test, but this is working for cnquery.

Reply:
https://github.com/mondoohq/cnquery/blob/main/cli/providers/providers.go#L377